### DISCUSSION TITLE
really big font

### DISCUSSION THREAD

**Discussion by Thomas6500** [olikraus/u8g2#1947] [Q&A]
Hi olikraus,

I really appreciate your work and intensity of replying all questions here. Keep up the good work, it helped me a lot, although I'm a beginner :)

I bought https://es.aliexpress.com/item/1005004252465006.html?spm=a2g0o.productlist.0.0.2d6e3950i1Ym9r&algo_pvid=1195c6bb-851a-4490-9030-b56a6c5f59d4&algo_exp_id=1195c6bb-851a-4490-9030-b56a6c5f59d4-0&pdp_ext_f=%7B%22sku_id%22%3A%2212000028529183731%22%7D&pdp_npi=2%40dis%21EUR%214.72%213.77%21%21%211.44%21%21%402100bddf16622181010833233e4ae6%2112000028529183731%21sea&curPageLogUid=Q79LCAscm1BT

Now, I have an issue how i can use fonts like this
![imagen](https://user-images.githubusercontent.com/112777195/188283733-b39e34c6-b2d7-424a-8d47-a31db0a97dec.png)

I'm a beginner, just sayin', but I try to deal with instructions

**Comment by olikraus** [comment]
> how i can use fonts like this

Select a font from here:
https://github.com/olikraus/u8g2/wiki/fntlist99

For example this font:
![https://github.com/olikraus/u8g2/wiki/fntpic/u8g2_font_logisoso58_tn.png](https://github.com/olikraus/u8g2/wiki/fntpic/u8g2_font_logisoso58_tn.png)

Then you remember the name of the font, here: u8g2_font_logisoso58_tn

Finally you use the font as argument for the u8g2.setFont() function, like this: `u8g2.setFont(u8g2_font_logisoso58_tn);`.

**Reply by olikraus** [reply]
Note: The above selected font only contains the above visible numbers and symbols (to reduce the required flash memory). Any attempt to print something like "abc" will not work.
